modules/sync/syncs/httpsynchronization.h

```diff
@@ -110,6 +117,16 @@ class HttpSynchronization : public ResourceSynchronization {
 
     // The thread that will be doing the synchronization
     std::thread _syncThread;
+
+    //The files that have already been synchronized
+    std::vector<std::string> _existingSyncedFiles;
+
+    //The files that have been synchronized this time
+    std::vector<std::string> _newSyncedFiles;
+
+    const std::string _ossyncVersionNumber = "1.0";
```

Comment:
```suggestion
    constexpr std::string_view _ossyncVersionNumber = "1.0";
```

Cheaper to create and compare